src/styles/blocks/cards/card.scss

```diff
@@ -0,0 +1,96 @@
+.card__wrapper {
+  margin: 0 16px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-image: url('/src/images/imac.jpeg');
```

Comment:
you don't need to use background-image in this case. It is best used for the background of

Reply:
↓